tests/test_basic.py

```
@@ -1,25 +1,33 @@
+    """
+    Test import of package_maximizer module
+    """
     try:
         import package_maximizer
 
         assert hasattr(package_maximizer, "__version__")
-        assert package_maximizer.__version__ == "0.1.0"
+        assert package_maximizer.__version__ == "0.2.0"
     except ImportError:
-        assert False, "Не удалось импортировать package_maximizer"
+        assert False, "Failed to import package_maximizer"
```

Ruff (0.16.2)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)